Libraries/Animated/src/AnimatedImplementation.js

```diff
@@ -169,12 +279,28 @@ class TimingAnimation extends Animation {
     this._duration = config.duration !== undefined ? config.duration : 500;
     this._delay = config.delay !== undefined ? config.delay : 0;
     this.__isInteraction = config.isInteraction !== undefined ? config.isInteraction : true;
+    this._useNativeDriver = !!config.useNativeDriver;
+  }
```

Comment:
looks like comments has been left on incorrect lines by the bot. I ended up running eslint locally to get the correct line numbers and fixed the issues cc @mkonicek